code/src/components/GameBoard.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { nextStage } from 'reducers/labyrinth';
+import styled from 'styled-components/macro';
+import TypeWriterEffect from 'react-typewriter-effect';
+import sleepingPotato from 'assets/sleeping-potato.gif';
+import { Container } from './styling/GlobalWrappers';
+
+const GameBoard = () => {
+  const dispatch = useDispatch();
+  const { description, actions, coordinates } = useSelector((store) => store.labyrinth.stage);
+  const userName = useSelector((store) => store.labyrinth.username);
+
+  const onDirectionButtonClick = (type, direction) => {
+    dispatch(nextStage(type, direction));
+  };
+
+  const onRestartButtonClick = () => {
+    return window.location.reload();
+  };
+
+  const winner = () => {
+    return (
+      <>
+        <WinnerWrapper>
+          <div>
+            <h2>YOU WIN!</h2>
+            <img src={sleepingPotato} alt="You win" />
+            <p>
+              Good job <span>{userName.toUpperCase()}</span>, you managed to get Potato back home
+              safely.
+            </p>
+          </div>
+        </WinnerWrapper>
+        <RestartBtn type="button" onClick={() => onRestartButtonClick()}>
+          RESTART
+        </RestartBtn>
+      </>
+    );
+  };
+  return (
+    <>
+      <Top hide={coordinates === '1,3'}>
+        <ButtonAndCompass>
+          <RestartBtn type="button" onClick={() => onRestartButtonClick()}>
+            RESTART
+          </RestartBtn>
+        </ButtonAndCompass>
+        <p>Player: {userName}</p>
+      </Top>
+      <Container>
+        <Description>
+          <TypeWriterEffect
+            startDelay={100}
+            cursorColor="#00cf00"
+            text={description}
+            typeSpeed={70}
+            hideCursorAfterText />
+        </Description>
+        <MazeImgContainer hide={coordinates === '1,3'}>
+          <ImgNavigation>
+            {actions
+              && actions.map((item) => {
+                return (
+                  <ButtonWrapper key={item.direction}>
+                    <button
+                      type="button"
+                      onClick={() => onDirectionButtonClick(item.type, item.direction)}>
+                      GO {item.direction.toUpperCase()}
+                    </button>
+                  </ButtonWrapper>
+                );
+              })}
+          </ImgNavigation>
+        </MazeImgContainer>
+        <ActionsWrapper>
+          {actions
+            && actions.map((item) => {
+              return (
+                <div key={item.description}>
+                  <Pdirection>WHAT DO YOU SEE TO THE {item.direction.toUpperCase()}:</Pdirection>
+                  <p>{item.description}</p>
+                </div>
+              );
+            })}
+        </ActionsWrapper>
+        {coordinates === '1,3' && winner()}
+      </Container>
+    </>
+  );
+};
+
+export default GameBoard;
+
+const RestartBtn = styled.button`
```

Comment:
We see that you are consistent and you have all your styled components in the same file. You could consider having a separate file for the styling of the styled components for cleaner code. You could also consider using styled-components to create the elements, such as the h2, div etc.